source/DSCResources/DSC_CMSoftwareUpdatePoint/DSC_CMSoftwareUpdatePoint.psm1

```diff
@@ -0,0 +1,458 @@
+                                            $anonymousWsus = $true
+                                        }
+                                        else
+                                        {
+                                            $anonymousWsus = $false
+                                        }
+                                    }
+                'WSUSIISPort'       {$wsusIis = $supProp.Value}
+                'WSUSIISSSLPort'    {$wsusIisSsl = $supProp.Value}
+            }
+        }
+        if (($internet) -and ($intranet))
+        {
+            $connectionType = 'InternetAndIntranet'
+        }
+        else
+        {
+            $connectionType = $internet + $intranet
+        }
+
+        $status = 'Present'
+    }
+
+    else
+    {
+        $status = 'Absent'
+    }
+
+    return @{
+        SiteServerName                = $SiteServerName
+        SiteCode                      = $SiteCode
+        AnonymousWSUSAccess           = $anonymousWsus
+        ClientConnectionType          = $connectionType
+        EnableCloudGateway            = $cloudGateway
+        UseProxy                      = $useProxyGeneral
+        UseProxyForAutoDeploymentRule = $useProxyForADR
+        WSUSAccessAccount             = $accessAccount
+        WSUSIISPort                   = $wsusIis
+        WSUSIISSSLPort                = $wsusIisSsl
+        WSUSSSL                       = $enableSsl
+        Ensure                        = $status
+    }
+}
+
+ <#
+    .SYNOPSIS
+        This will set the desired state.
+
+    .PARAMETER SiteCode
+        Specifies a site code for the Configuration Manager site.
+
+    .PARAMETER SiteServerName
+        Specifies the Site Server to install or configure the role on.
+
+    .PARAMETER AnonymousWSUSAccess
+        Indicates that the software update point allows anonymous access. Mutually exclusive with WSUSAccessAccount.
+
+    .PARAMETER ClientConnectionType
+        Specifies the type of the client connection. The acceptable value are Internet,
+        Intranet, and InterneAndIntranet.
+
+    .PARAMETER EnableCloudGateway
+        Specifies if a cloud gateway is to be used for the software update point.
+        When enabling the cloud gateway, the client connectiontype must be either Internet or InterneAndIntranet.
+        When enabling the cloud gateway, SSL must be enabled.
+
+    .PARAMETER UseProxy
+        Indicates whether a software update point uses the proxy configured for the site system server.
+
+    .PARAMETER UseProxyForAutoDeploymentRule
+        Indicates whether an auto deployment rule can use a proxy.
+
+    .PARAMETER WSUSAccessAccount
+        Specifies an account used to connect to the WSUS server. When not used, specify the AnonymousWSUSAccess parameter.
+
+        If specifying an account the account must already exist in
+        Configuration Manager.  This can be achieved by:
+
+        $secure = ConvertTo-SecureString -String "Password" -AsPlainText -Force
```

Comment:
you could reference the resource in here vs the command line /shrug

Reply:
I went ahead and changed both here and test over to say it can be achieved by using the CMAccounts Resource.